hooks.lua

```diff
@@ -56,12 +51,13 @@ function OnPlayerLeftClick(a_Player, a_BlockX, a_BlockY, a_BlockZ, a_BlockFace,
 	end
 	
 	Collection:Finish()
-	
+
 	if (g_Config.ReplantSapling) then
-		World:QueueSetBlock(a_BlockX, a_BlockY, a_BlockZ, E_BLOCK_SAPLING, GetSaplingMeta(BlockType, BlockMeta), g_Config.SaplingPlantTime)
+			math.randomseed(os.time())
```

Comment:
I don't think it's needed to change the seed every time.

Reply:
I've put this in `init.lua`